Demo/src/main/java/com/paypal/android/api/services/Merchant.kt

```diff
@@ -0,0 +1,8 @@
+package com.paypal.android.api.services
+
+enum class Merchant(val baseUrl: String) {
```

Comment:
Another nitpick on naming - just because `Merchant` is vague IMO

```suggestion
enum class MerchantAccountType(val baseUrl: String) {
```

Reply:
Yeah that makes sense. We may go with `MerchantIntegration` to avoid baking `Type` into the name. I like this snippet from [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#name-according-to-roles). We aren't in Swift, but that naming guideline applies to a lot of languages I feel.